packages/frontendmu-adonis/inertia/css/app.css

```
@@ -17,6 +17,19 @@
   --color-verse-800: #0c1a3d;
   --color-verse-900: #060d1f;
   --color-verse-950: #020617;
+
+  /* Coral accent — used sparingly for live/action moments and section labels */
+  --color-coral-soft: oklch(94% 0.04 40);
+  --color-coral: oklch(68% 0.17 35);
+  --color-coral-strong: oklch(62% 0.19 32);
+
+  /* Mono font stack */
+  --font-mono:
+    'JetBrains Mono', ui-monospace, 'SF Mono', Menlo, Consolas, monospace;
```

Stylelint (17.7.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)